mobile/src/assets/images/vendors/coursera.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="50" height="7" viewBox="0 0 50 7" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
let's remove everything related to coursera, for now we'll have only udemy integration